src/options/options.html

```diff
@@ -179,6 +179,13 @@ <h1 data-i18n="__MSG_titleAddonBehavior__">Addon behavior</h1>
 						</div>
 					</li>
 
+					<li>
+						<div class="line">
+							<input class="setting save-on-change" type="checkbox" id="disableContextMenu" name="disableContextMenu">
+							<label data-i18n="__MSG_optionDisableContextMenu__" for="disableContextMenu">Disable the context menu item for selected text</label>
```

Comment:
Great you added that option and i18n entry. However please also provide the translation string in the `message.json` file, so people can translate it. It is enough to provide the English translation, of course, so you can just copy and paste the string from here.
This ensures, that text snippet is ready for translation and can be localized.

For more information, please [see the MDN guide on how to localize WebExtensions](https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization#Providing_localized_strings_in__locales) and [our internationalization (i18n) guide](https://github.com/TinyWebEx/common/blob/master/CONTRIBUTING.md#internationalisation-i18n).